airbyte-integrations/connectors/source-sftp-bulk/integration_tests/integration_test.py

```diff
@@ -36,16 +34,46 @@ def docker_client():
     return docker.from_env()
 
 
+@pytest.fixture(scope="session")
+def path_to_mount_to_sftp(tmp_path_factory):
+    """
+    Copy the integration_test/files folder to a tmp dir which is shared with the docker host.
+    """
+    path = tmp_path_factory.mktemp("sftp_files")
+    shutil.copytree(f"{os.getcwd()}/integration_tests/files", path, dirs_exist_ok=True)
+    return path
+
+
+@pytest.fixture(scope="session")
+def key_pair():
+    return generate_ssh_keys()
+
+
+@pytest.fixture(scope="session")
+def public_key(key_pair):
+    return key_pair[1]
+
+
+@pytest.fixture(scope="session")
+def private_key(key_pair):
+    return key_pair[0]
+
+
+@pytest.fixture(scope="session")
+def public_key_path(tmp_path_factory, public_key):
+    public_key_path = tmp_path_factory.mktemp("ssh") / "id_rsa.pub"
+    public_key_path.write_text(public_key)
+    return public_key_path
```

Comment:
@sherifnada in the `airbyte-ci` context the container running the integration test is bound to a sidecar `dockerd` container used as the `DOCKER_HOST`. You can consider this sidecar container as a docker machine: it's this container that manages the container the client (the integration test) run. 
In `airbyte-ci` we only make the `/tmp` folder shared between the client and the docker host. As these test need to mount file test the `sftp` container I added these fixture to move these files a `/tmp` dir.